src/helpers.c

```diff
@@ -223,17 +223,17 @@ static bool validate_tracking_dir(const char *state_dir)
 	return true;
 }
 
-int create_state_dirs(const char *state_dir_path)
+int create_basic_state_dirs(const char *state_dir_path)
 {
 	int ret = 0;
 	unsigned int i;
 	char *dir;
 #define STATE_DIR_COUNT (sizeof(state_dirs) / sizeof(state_dirs[0]))
-	const char *state_dirs[] = { "delta", "staged", "download", "telemetry", "bundles", "3rd_party" };
+	const char *state_dirs[] = { "delta", "staged", "download", "telemetry", "bundles" };
```

Comment:
Why do we need to create the 3rd_party separate from the rest?

Reply:
because the state directories of a 3rd-party repo don't need that directory, so this way we can reuse the same function to create the state directories with these repos

Reply:
having said that it really doesn't hurt if that extra dir is there... I'll leave this as it was in the next update